test/utils/utils.test.js

```diff
@@ -31,6 +30,13 @@ describe('Utils', () => {
     delete window.hlx;
   });
 
+  it('fetches with cache param', async () => {
+    window.fetch = mockFetch({ payload: true });
+    const mockUrl = './mocks/taxonomy.json';
+    const resp = await customFetch({ resource: mockUrl, withCacheRules: true });
```

Comment:
Nit: I think we can give the string value directly to the `resource` key. That would still be within the linter chars limit for 1 line.
```suggestion
    const resp = await customFetch({ resource: './mocks/taxonomy.json', withCacheRules: true });
```